src/plugins/Debug.ts

```diff
@@ -24,6 +28,18 @@ class Debug extends GenericPlugin {
     const { container } = this.ad;
 
     this.debugOverlay = insertElement('div', { style: OVERLAY_STYLE }, container);
+    googletag.cmd.push(() => {
+      googletag.pubads().addEventListener(
+        'slotRenderEnded',
+        (event: googletag.events.SlotRenderEndedEvent) => {
+          this.insertMetadata(event);
+        },
+      );
+    });
+  }
+
+  public afterCreate() {
```

Comment:
Adding after create will give the second time stamp from requesting creation to created to rendered